contrib/xfce4-mixer/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "xfce4-mixer"
+pkgver = "4.18.1"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "gettext-devel",
+    "glib-devel",
+    "gmake",
+    "intltool",
+    "libtool",
+    "pkgconf",
+    "xfce4-dev-tools",
+]
+makedepends = [
+    "alsa-lib-devel",
```

Comment:
we are generally not interested in enabling alsa backends in stuff that supports pulse/pw